homeassistant/components/homematic/entity.py

```diff
@@ -150,6 +162,8 @@ def _subscribe_homematic_events(self):
                         channel = channels[0]
                     else:
                         channel = self._channel
+                    # Remember the channel for this attribute to ignore invalid events later
+                    self._channel_map[node] = self._address + ":" + str(channel)
```

Comment:
Maybe we should remove our complex (current) subscription handling and just listen to all events and filter that like you do now with this channel map?

Reply:
That would be a possibility. If we do that I would go even one step further and _whitlist_ events live _PRESS\_LONG_ etc.. If we did that we wouldn't have to add support for these events manually, they would just always work and fire the events.

Reply:
We did not speak about the same :)